src/components/case-study-single/Explanation.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Explanation = ({ explanation }) => {
+  const explanationElements = explanation.map((item, index) => {
+    // console.log(typeof item);
```

Comment:
Again a couple of things that have been commented out look like they can be removed

Reply:
Thanks, this study is incomplete at the moment